Game/Resources.json

```diff
@@ -1,5 +1,5 @@
 {
-	"activeScene" : "Resources/Super Scene/Super Scene",
+	"activeScene" : "Resources/elevatorScene/elevatorScene_01",
```

Comment:
Would prefer if the startup-scene always is Super Scene on master.

Reply:
This is not the startup scene, but the active scene (scene open in editor).

Reply:
That's what I meant, starting up the editor ;)